extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java

```diff
@@ -41,13 +55,20 @@ public void testGetComments() throws IOException, ExtractionException {
             result = findInComments(comments, "s1ck m3m3");
         }
 
-        assertTrue(result);
+        return result;
+
     }
 
     @Test
     public void testGetCommentsFromCommentsInfo() throws IOException, ExtractionException {
+        assertTrue(getCommentsFromCommentsInfoHelper(urlYT));
+        assertTrue(getCommentsFromCommentsInfoHelper(urlInvidious));
+        assertTrue(getCommentsFromCommentsInfoHelper(urlInvidioush));
+    }
+
+    private boolean getCommentsFromCommentsInfoHelper(String url) throws IOException, ExtractionException {
         boolean result = false;
-        CommentsInfo commentsInfo = CommentsInfo.getInfo("https://www.youtube.com/watch?v=D00Au7k3i6o");
+        CommentsInfo commentsInfo = CommentsInfo.getInfo(url);
```

Comment:
``` Java
assertTrue("what the fuck am i doing with my life".equals(commentsInfo.getName()));
```
Can you convert this to `assertEquals()` please?

Reply:
done

Reply:
thanks